templates/server/locations/proxy.erb

```diff
@@ -46,4 +46,12 @@
 <% if @proxy_cache_key -%>
     proxy_cache_key         <%= @proxy_cache_key %>;
 <% end -%>
+<% if @proxy_cache_bypass && Array(@proxy_cache_bypass).size > 0 -%>
```

Comment:
Is the size check needed?

Reply:
Initially when I've added theses features I've started from the 0.5.0 version which was published on forge and followed the same coding style, that's why there is this check I believe: 
I totally agree with you it seems useless now.